code/modules/medical/cloning.dm

```diff
@@ -429,6 +429,26 @@
 	occupant.forceMove(exit)
 	icon_state = "pod_0"
 	eject_wait = FALSE //If it's still set somehow.
+	//do early ejection damage
+	var/completion = 10*((occupant.health + 100) / (heal_level + 100)) //same way completion is calculated for examine text, but out of 10 instead of 100
+	var/damage_rolls = 10 - round(completion) - (round(resource_efficiency) - 1) // 1 roll for each 10% missing, each improved pair of manipulators reduces one roll
+	var/hits = 0
+	while(damage_rolls > 0)
+		if(prob(25))//each roll has a 25% chance to give the occupant a bad time
+			hits++
+		damage_rolls--
+	//apply the damage
+	var/mob/living/carbon/human/H = occupant
+	while(hits>0)
+		if (hits>=4)
+			qdel(pick(H.internal_organs - H.internal_organs["brain"]))
```

Comment:
`internal_organs` will never contain the string `brain`, you might be thinking of `internal_organs_by_name`

Reply:
you're right